indico/modules/events/abstracts/util.py

```diff
@@ -379,3 +382,15 @@ def get_events_with_abstract_persons(user, dt=None):
     for person in query:
         data[person.event_id].add('abstract_person')
     return data
+
+
+def filter_field_values(fields, user, managers, submitter):
+    if user in managers:
```

Comment:
Just like above, I would check for management area instead of management rights

Reply:
And here also, wouldn't it be confusing for a manager to see different fields on display and different on manage?

Reply:
yeah, let's stay with your logic and  just fix the way of checking for management rights ;)